speed_expr/speed_expr.cpp

```diff
@@ -1,24 +1,70 @@
-#include <string>
+#include <fstream>
 #include <iostream>
 
-#include "../util/configuration.h"
-#include "../util/insertion_mgr.h"
+#include <graph_zeppelin_common.h>
+#include "../util/shell_exec.h"
+
+std::string cat(std::string filename) {
+  return shell_exec("cat " + filename);
+}
 
 int main(int argc, char** argv) {
-  if (argc != 3) {
-    std::cout << "Incorrect number of arguments. "
-                 "Expected two but got " << argc-1 << std::endl;
-    std::cout << "Arguments are: input_stream, output_file" << std::endl;
-    exit(EXIT_FAILURE);
+  if (argc < 3) {
+    std::clog << "Correct usage is\n"
+      "\t/path/tp/speed_experiment csv_output_file input_file [input files+]" <<std::endl;
```

Comment:
typo

Reply:
Why clog?

Reply:
idk, it’s similar to cerr but buffered. do you like cerr more?

Reply:
Doesn't matter to me. I just didn't know what clog was :)